src/cljs/main/broadfcui/config.cljs

```diff
@@ -20,7 +20,7 @@
         optional {"cromwellVersion" :string "isDebug" :boolean "shibbolethUrlRoot" :string
                   "submissionStatusRefresh" :integer "userGuideUrl" :string
                   "workflowCountWarningThreshold" :integer "billingGuideUrl" :string
-                  "callCachingGuideUrl" :string}
+                  "callCachingGuideUrl" :string "alertsJsonUrl" :string}
```

Comment:
(for the record I'm not going to change it until I figure out why things that are way more important such as `shibbolethUrlRoot` are in the optional section)